src/microbe_stage/MicrobeStage.cs

```diff
@@ -1108,6 +1111,12 @@ private void OnPlayerEngulfedByHostile(Entity player, Entity hostile)
         }, this);
     }
 
+    [DeserializedCallbackAllowed]
+    private void OnPlayerEjectedFromHostileEngulfer(Entity player)
+    {
+        OnCanEditStatusChanged(player.Get<OrganelleContainer>().AllOrganellesDivided);
```

Comment:
As this can be called by a thread, this needs to invoke like the other callbacks. Also there is extra logic in OnPlayerReproductionStatusChanged that this goes around.

Here's my suggestion how to fix this:
```suggestion
        // Re-check the reproduction status with the normal reproduction status check
        OnPlayerReproductionStatusChanged(player.Get<OrganelleContainer>().AllOrganellesDivided);
```